test/vmtests/vmtests/utils/libvirt_utils.py

```diff
@@ -117,9 +232,10 @@ def _add_disk_xml(
     device_type: str,
     image_type: str,
     bus_type: str,
+    device_prefix: str,
```

Comment:
You can probably derive `device_prefix` from `bus_type`.

vd => virtio device
sd => sata device